src/Components/Authentication/Desktop/LoginForm.tsx

```diff
@@ -16,6 +16,7 @@ import PasswordInput from "Components/PasswordInput"
 import QuickInput from "Components/QuickInput"
 import { Formik, FormikProps } from "formik"
 import React, { Component } from "react"
+import { Grepcaptcha, GrepcaptchaAction } from "Utils/Grepcaptcha"
```

Comment:
Can probably drop the "g" from the recaptcha -- seems to be encoding implementation details ("recaptcha" is a universal thing, g + recaptcha is some other thing, plus the universal). If we wanted to use some other recaptcha implementation, we should be able to swap it out silently in the background. ("grecapta" is also a bit weird to type 😄)